packages/espressocash_app/lib/features/outgoing_split_key_payments/src/bl/repository.dart

```diff
@@ -178,6 +181,7 @@ extension on OSKPStatusDto {
     final link3 = row.link3?.let(Uri.tryParse);
     final cancelTx = row.cancelTx?.let(SignedTx.decode);
     final cancelTxId = row.cancelTxId;
+    final resolvedAt = row.resolvedAt ?? DateTime.now();
```

Comment:
Don't use `DateTime.now()` as a fallback. Otherwise, it will be different on every loading.

It's better to make the field nullable in that case (but still required), and in the future, when all the clients are migrated, we can move it back to non-nullable.